ftl13.dme

```diff
@@ -1399,6 +1382,33 @@
 #include "code\modules\food_and_drinks\recipes\tablecraft\recipes_soup.dm"
 #include "code\modules\food_and_drinks\recipes\tablecraft\recipes_spaghetti.dm"
 #include "code\modules\ftl\machinery\ftl_shieldgen.dm"
+#include "code\modules\ftl\machinery\computers\navigation.dm"
+#include "code\modules\ftl\misc\fdr.dm"
+#include "code\modules\ftl\misc\toys.dm"
+#include "code\modules\ftl\objectives\objective.dm"
+#include "code\modules\ftl\objectives\boarding\boarding.dm"
+#include "code\modules\ftl\objectives\boarding\init.dm"
+#include "code\modules\ftl\objectives\boarding\terminal.dm"
+#include "code\modules\ftl\objectives\boarding\gear\at_turret.dm"
+#include "code\modules\ftl\objectives\boarding\gear\gear.dm"
+#include "code\modules\ftl\objectives\boarding\gear\mines.dm"
+#include "code\modules\ftl\objectives\boarding\gear\misc.dm"
+#include "code\modules\ftl\objectives\boarding\gear\telecrystals.dm"
+#include "code\modules\ftl\objectives\boarding\outfit\clown.dm"
+#include "code\modules\ftl\objectives\boarding\outfit\generic.dm"
+#include "code\modules\ftl\objectives\boarding\outfit\nanotrasen.dm"
+#include "code\modules\ftl\objectives\boarding\outfit\outfit.dm"
+#include "code\modules\ftl\objectives\boarding\outfit\pirate.dm"
+#include "code\modules\ftl\objectives\boarding\outfit\solgov.dm"
```

Comment:
Boarding sohuldn't be in objectives

Reply:
@monster860 Why? It's a mission objective, it makes sense to me.